### ISSUE TITLE
Frontend: Project info is absent after page reloading

### ISSUE THREAD

**Issue by kgevorkyan** [analysis-dev/save-cloud#588]
At the initial state all fields are `N/A`, however DB contain all necessary information

![1](https://user-images.githubusercontent.com/26010098/155110357-c165f506-e2ed-4267-a67c-d59fea3f0c27.PNG)

After clicking to the `edit` button and then to `cancel`, information appears: 

![2](https://user-images.githubusercontent.com/26010098/155110362-e7304c6f-29e9-455c-8a12-5b795cbe5e91.PNG)

**Comment by petertrr** [comment]
That's because this form is not re-rendered after project is fetched and it stays with default placeholder data. All forms should be [controlled components](https://reactjs.org/docs/forms.html#controlled-components) in react, i.e. have inputs bound to state.

**Comment by kgevorkyan** [comment]
hm, seems that we break something in one of the latest PR, initially it worked fine

**Comment by petertrr** [comment]
Yeah, it must be because of #579, where I've removed unnecessary rerenders. I believe because edit form was rerendered, we could see correct data. I.e., visibly it was correct behavior but with incorrect implementation
